pkg/machine/wsl/machine.go

```diff
@@ -1105,8 +1105,15 @@ func (v *MachineVM) Set(_ string, opts machine.SetOptions) ([]error, error) {
 			v.UserModeNetworking = *opts.UserModeNetworking
 		}
 	}
+	err := v.writeConfig()
+	if err != nil {
+		setErrors = append(setErrors, err)
+	}
 
-	return setErrors, v.writeConfig()
+	if len(setErrors) > 0 {
+		return setErrors, setErrors[0]
+	}
+	return setErrors, nil
```

Comment:
What is the reason behind this chnage? Why is the same error returned twice?

Reply:
i am simply following what qemu did to make things consistent.  i think the questions are valid however but maybe for another time?